compiler/modules/types/table.bal

```diff
@@ -0,0 +1,36 @@
+// Implementation specific to basic type table.
+
+public function tableContaining(SemType memberType) returns SemType {
+    SubtypeData ro = subtypeData(memberType, UT_MAPPING_RO);
+    SubtypeData rw = subtypeData(memberType, UT_MAPPING_RW);
```

Comment:
Unfortunately, it's not this simple https://github.com/ballerina-platform/nballerina/issues/713#issuecomment-985316349